arangod/VocBase/LogicalCollection.h

```diff
@@ -338,6 +340,8 @@ class LogicalCollection: public LogicalDataSource {
   // Get a reference to this KeyGenerator.
   // Caller is not allowed to free it.
   inline KeyGenerator* keyGenerator() const { return _keyGenerator.get(); }
+  
+  transaction::CountCache& countCache() { return _countCache; }
```

Comment:
can't we move this into the CoordinatorCollection ? The LogicalCollection has too many members that are only used half the time